autograder/cli/users/remove.py

```diff
@@ -0,0 +1,23 @@
+import sys
+
+import autograder.api.users.remove
+
+def run(arguments):
+    result = autograder.api.users.remove.send(arguments, exit_on_error = True)
+
+    if (result['found-user']):
+        print("User removed.")
+    else:
+        print("User not found.")
```

Comment:
Go ahead and return 1 in this case.
(And since your returning, the structure will change a bit (probably just an if and no else).)

I'm a little on the fence about this,
but I think we want to return a non-zero exit status in this case.
I am taking my lead from `rm`, which returns non-zero if the dirent does not exist.
(Note that it will return 0 with `-f`, but we don't have that option here.)